webpack/webpack.common.js

```diff
@@ -425,112 +434,31 @@ module.exports = () => {
                         ],
                     },
                     {
-                        test: /\.html?$/i,
-                        exclude: /node_modules/,
-                        loader: Path.join(PROJECT_RELATIVE_NODE_MODULES_PATH, 'html-loader'),
-                        options: {
-                            esModule: false,
-                            minimize: {
-                                removeComments: false,
+                        test: /\.(woff2?|eot|ttf|otf)$/i,
+                        type: 'asset/resource',
+                        generator: {
+                            filename: 'fonts/[name].[contenthash][ext]',
+                        },
+                    },
+                    {
+                        test: /\.htm$/i,
```

Comment:
This change, instead of loading the template into the webpack bundler, will instead just map the relative path to the component import statement. This allows us to hijack how KOJS handles component registration